doc/_resource_types/acm.md

```diff
@@ -0,0 +1,33 @@
+### exist
+
+```ruby
+describe acm('example.com') do
+  it { should exist }
+end
+```
+
+### be_issued
+
+```ruby
+describe acm('example.com') do
+  it { should be_issued }
+end
+```
+
+### advanced
+
+`acm` can use `Aws::ACM::Types::CertificateDetail` resource (see http://docs.aws.amazon.com/sdkforruby/api/Aws/ACM/Types/CertificateDetail.html).
```

Comment:
This is not `advanced` 

##  What is advanced?

use `aws_resource` like ec2 type.

see https://speakerdeck.com/k1low/inside-of-awspec?slide=27